packages/component-library/components/Notification/_styles.scss

```diff
@@ -218,7 +243,11 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
   }
 
   %ca-notification---global & {
-    @include kz-typography-paragraph-body;
+    font-weight: $kz-var-typography-paragraph-body-font-weight;
+    font-size: $kz-var-typography-paragraph-body-font-size;
+    line-height: $kz-var-typography-paragraph-body-line-height;
+    letter-spacing: $kz-var-typography-paragraph-body-letter-spacing;
+    top: 0;
```

Comment:
Is this the right way to "inherit baseline" ?

Reply:
I would import
`@kaizen/deprecated-component-library-helpers/styles/type.scss` and use `@include inherit-baseline`